### PR TITLE
#166 make imports sorting optional during organize imports

### LINKED ISSUE #166: Remove unused imports without sorting
#### Feature: Organize imports only by removing unused ones

Enable an option for cleaning the imports without changing their order.

#### Reason

Sometimes I want to preserve the order of the imports for a reason (i.e. keep same kind of imports together). It would be great to have the ability to do that kind of organization.

### PR DESCRIPTION
**Pull request by grzegorz-jazurek**
I added `disableImportsSorting` setting which resolve #166

I have problems with unit tests, because right now they don't pass, and I supposed that providing structure for configuration mock services is part of refactor which you mention.

### CODE REVIEW

**Comment by grzegorz-jazurek** [comment]
sorry I didn't notice `feature/rework-extension` branch. If you like I'll try to apply my changes to this branch

**Comment by buehler** [comment]
Hey @grzegorz-jazurek 
I'm currently attending @ the JSConf in Berlin :-)

No worries about the branch, I'll merge your branch, as soon as I reviewed it, an hopefully the tests are running 👍 

Cheers

**Comment by grzegorz-jazurek** [comment]
thanks, have a nice conference!

File: package.json [hunk 1]
```diff
@@ -190,6 +190,11 @@
           "default": "TopOfFile",
           "description": "Defines the location of new imports."
         },
+        "typescriptHero.resolver.disableImportsSorting": {
+          "type": "boolean",
+          "default": false,
```
Comment (buehler) [inline package.json]: default should be true, to stick with the actual behaviour of the extension

Comment (buehler) [inline package.json]: or don't you think? the user can overwrite it with the vscode user settings

Comment (grzegorz-jazurek) [inline package.json]: You're right, I'll change it

**Comment by buehler** [comment]
the rewrite will contain this, and will then implement #102 to replace this

**Pull request merged**